### PR TITLE
{cae}[intel/2019b] OpenFOAM-Extend v4.1-20191120 w/ Python 2.7.16

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1981~~ + ~~https://github.com/easybuilders/easybuild-framework/pull/3225~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2507.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ad129005d730e228c2644fa012013276 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c403948ac85572b035ad4c6f40f8ecc6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso-3 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/f496f19f687b39a73f78622636f37b1c for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/d98b2ceebc882623169edc9a4f8a888e for a full test report.

**Comment by migueldiascosta** [comment]
it seems `ParMGridGen` isn't available at that location anymore (?)

**Comment by boegel** [comment]
@migueldiascosta We could start using http://sourceforge.mirrorservice.org/f/project/fo/foam-extend/ThirdParty/ instead?

**Comment by boegel** [comment]
@migueldiascosta fixed in #10019

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/4ab6b276215f65e88f91b83df451b1f3 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/20214fc86a84a43a29ff67a0c417dc10 for a full test report.

File: easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-4.1_comp-mpi.patch [hunk 1]
```diff
@@ -0,0 +1,50 @@
+diff -ruN foam-extend-4.1.orig/wmake/rules/linux64Icc/mplibEASYBUILDMPI foam-extend-3.0/wmake/rules/linux64Icc/mplibEASYBUILDMPI
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-4.1_comp-mpi.patch]: author and description?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3128.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/6e649782213552fbf43e6d3ef539a027 for a full test report.

**Pull request merged**